demo/ios/OrcaDemo/OrcaDemo/ContentView.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  Copyright 2024 Picovoice Inc.
+//  You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+//  file accompanying this source.
+//  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+//  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+//  specific language governing permissions and limitations under the License.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    @StateObject var viewModel = ViewModel()
+    @State private var text = ""
+
+    let activeBlue = Color(red: 55 / 255, green: 125 / 255, blue: 1, opacity: 1)
+    let dangerRed = Color(red: 1, green: 14 / 255, blue: 14 / 255, opacity: 1)
+    let lightGray = Color(red: 247 / 255, green: 247 / 255, blue: 247 / 255, opacity: 1)
+
+    var body: some View {
+        let interactionDisabled =
+            !viewModel.errorMessage.isEmpty || viewModel.state == UIState.PROCESSING
+            || viewModel.state == UIState.INIT || text.isEmpty
+        GeometryReader { _ in
+            VStack(spacing: 10) {
+                GeometryReader { geometry in
+                    VStack {
+                        ScrollView {
+                            TextEditor(text: $text)
+                                .transparentScrolling()
+                                .padding()
+                                .frame(minWidth: 0,
+                                       maxWidth: .infinity,
+                                       minHeight: geometry.size.height,
+                                       maxHeight: .infinity)
+                                .font(.title3)
+                                .background(lightGray)
+                                .onChange(of: text) { _ in
+                                    text = String(text.prefix(Int(viewModel.maxCharacterLimit)))
+                                }
+                        }
+
+                        Text("\(text.count) / \(viewModel.maxCharacterLimit)")
+                            .font(.footnote)
+                            .frame(maxWidth: .infinity, alignment: .trailing)
+                            .foregroundColor(Color.gray)
+                    }
+                }
+
+                if viewModel.state == .INIT || viewModel.state == .READY {
+                    Text("Enter any text to be synthesized")
```

Comment:
Any way you could move this as a hint to the text area? It makes it more obvious that the textbox is where you'll be entering the text